test/integration/hook.cors_csrf.test.js

```diff
@@ -86,8 +86,9 @@ describe('CORS and CSRF ::', function() {
             url: 'test',
           }, function(err, response) {
             if (err) return done(new Error(err));
+            body = response.body.split(',').sort().join(',');
             assert.equal(response.statusCode, 200);
-            assert.equal(response.body, "GET,POST,PUT,HEAD,DELETE,TRACE,COPY,LOCK,MKCOL,MOVE,PURGE,PROPFIND,PROPPATCH,UNLOCK,REPORT,MKACTIVITY,CHECKOUT,MERGE,M-SEARCH,NOTIFY,SUBSCRIBE,UNSUBSCRIBE,PATCH,SEARCH,CONNECT", require('util').format('Unexpected HTTP methods:  "%s"', response.body));
+            assert.equal(body, 'CHECKOUT,CONNECT,COPY,DELETE,GET,HEAD,LOCK,M-SEARCH,MERGE,MKACTIVITY,MKCOL,MOVE,NOTIFY,PATCH,POST,PROPFIND,PROPPATCH,PURGE,PUT,REPORT,SEARCH,SUBSCRIBE,TRACE,UNLOCK,UNSUBSCRIBE', require('util').format('Unexpected HTTP methods:  "%s"', response.body));
             done();
```

Comment:
No, it won't. Coz now it always sorts the list of methods before the assertion.
I just checked it on a regular node (v0.10.37) - all tests pass just fine.

As for the `_.contains` - I just don't see how exactly it'd fit here :)